EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79047
+upload-4ever.com#?#div[class^="my-"]:has(> ins.adsbygoogle)
```

Comment:
I'd use `:upward` `upload-4ever.com#?#div[class^="my-"] > ins.adsbygoogle:upward(1)`

Reply:
Same thing.

Reply:
We are all right in our own perspective.

Reply:
Not the same but in this case :has writing is fine as the number of nodes to evaluate is 2 vs. 3 against :upward writing.

Reply:
I know this, but after some cases, I prefer to use `:has` only when necessary.

Reply:
@wout-codes If you wonder, I talked about performance than what are hidden.